test/playwright/e2e/global-audio.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test, expect, Page, Locator } from '@playwright/test'
+
+import { sleep } from '~~/test/playwright/utils/navigation'
+
+const getNthAudioRow = async (page: Page, num: number) => {
+  const rowSelector = `[aria-label*="Audio Player"] >> nth=${num}`
+  const nthAudioRow = await page.locator(rowSelector).locator('article')
+  expect(nthAudioRow).toHaveAttribute('status', 'paused')
+  return nthAudioRow
+}
+
+const play = async (audioRow: Locator) => {
+  await audioRow.locator('button[aria-label="Play"] >> visible=true').click()
+  expect(audioRow).toHaveAttribute('status', /(loading|playing)/)
+}
+
+test.describe('global audio', () => {
```

Comment:
I think we could also add a test case where we click play and then pause on the search page, and then check that the audio **isn't** playing on the detail page.

Reply:
I had that test written but didn't like how I named it. I just added it with a better name :)